system_tests/monitoring.py

```diff
@@ -207,9 +207,14 @@ def _query_timeseries_with_retries():
             def _has_timeseries(result):
                 return len(list(result)) > 0
 
+            def _unknown_metric(result):
+                return ('The provided filter doesn\'t refer to any known '
+                        'metric.'in result.message)
```

Comment:
1. Space between the string and `in`
1. Please define `_unknown_metric` as a global and make the "The provided filter" string a global
1. Is there no valuable field in the JSON payload of the error response? (See #2414 about issues parsing the errors out of the payload)